langchain/schema.py

```diff
@@ -86,6 +88,15 @@ def type(self) -> str:
         """Type of the message, used for serialization."""
         return "human"
 
+class HumanFeedbackMessage(BaseMessage):
```

Comment:
im not sure this is general - would prefer to only have this in experimental

Reply:
Thanks for your feedback, I fixed it. The HumanFeedbackMessage class is not needed since using HumanMessage for the feedback also work.